lib/src/components/NxNavigationSidebar/NxNavigationSidebar.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent } from 'react';
+import classnames from 'classnames';
+
+import NxButton from '../NxButton/NxButton';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { Props, propTypes } from './types';
+
+import './NxNavigationSidebar.scss';
+
+const defaultLogo = require('../../assets/img/SON_hexagon_cropped.svg');
+
+const NxNavigationSidebar: FunctionComponent<Props> = function NxNavigationSidebar(props) {
+  const {
+    isOpen,
+    children,
+    className,
+    toggleOpenIcon,
+    toggleCloseIcon,
+    onToggleClick,
+    logoImg,
+    logoAltText,
+    logoLink
+  } = props;
+
+  const classes = classnames(className, 'nx-page-sidebar-operable', {
+    'open': isOpen,
+    'closed': !isOpen
+  });
+
+  const toggleButtonIcon = isOpen ? toggleOpenIcon : toggleCloseIcon;
+  const toggleButton = (
+    <NxButton variant="icon-only"
```

Comment:
Needs an `aria-label` for screenreaders.

Reply:
> Needs an aria-label for screenreaders.

Or alternatively a tooltip

Reply:
> Needs an `aria-label` for screenreaders.

Like this? 5f59c62f5ff38da30bcaf61d4b3e8dffd1128d69

Reply:
> Like this? 5f59c62

Something clearer than that would be better.  Like "Collapse Sidebar" and "Expand Sidebar"

Reply:
> > Like this? [5f59c62](https://github.com/sonatype/sonatype-react-shared-components/commit/5f59c62f5ff38da30bcaf61d4b3e8dffd1128d69)
> 
> Something clearer than that would be better. Like "Collapse Sidebar" and "Expand Sidebar"

Improved in f1ba23a516e615babc21c75e483a1b15f06a8d25